tests/test_configApdbLoader.py

```diff
@@ -41,15 +43,17 @@ class DummyConfigurable:
         registry.register("bar", ApdbSql, ConfigClass=ApdbSqlConfig)
         return registry
 
-    @staticmethod
-    def _dummyApdbConfig():
-        config = ApdbSqlConfig()
-        config.db_url = "sqlite://"     # in-memory DB
-        return config
+    def _dummyApdbConfig(self):
+        return ApdbSql.init_database(db_url=self.db_url)
 
     def setUp(self):
+        self.tempdir = tempfile.mkdtemp()
+        self.db_url = f"sqlite:///{self.tempdir}/apdb.sqlite3"
```

Comment:
Why this change? Does `sqlite://` fail with the new system?

Reply:
It will fail if you actually try to use Apdb. Schema is created in `init_database` but you get back config instance instead of connection handle, so next time you instantiate Apdb you have a new connection to the database which in case of `:memory:` database means completely fresh database (no tables). In-memory sqlite database cannot be used with Apdb anymore (which is a good thing because it's too much trouble for some access patterns).